django/db/models/base.py

```diff
@@ -40,6 +37,16 @@
 from django.utils.version import get_version
 
 
+class Deferred(object):
+    def __repr__(self):
+        return '<Deferred field>'
+
+    def __str__(self):
+        return '<Deferred field>'
```

Comment:
I remember issues with objects returning `unicode` in `__str__` and `__repr__` on Python 2. Better be extra safe and return `str('<Deferred field>')` in both cases.